app.js

```diff
@@ -444,20 +455,39 @@ if (overlayCast) {
   searchPorts();
 }
 
+const handleDeeplink = async (url) => {
+  const urlObject = url.replace('sttm-desktop://', '').split('?');
+  if (urlObject[0].includes('login')) {
+    const loginData = new URLSearchParams(`?${urlObject[1]}`);
+    const token = loginData.get('token');
+    if (token) {
+      await keytar.setPassword('sttm-desktop', 'userToken', token);
+      mainWindow.webContents.send('userToken', token);
+    }
```

Comment:
Instead of just putting an `if` here, I would suggest you put a valid exception handling with `try`.

Reply:
Added try catch block inside the if statement.